pkg/kubectl/cmd/delete.go

```diff
@@ -123,17 +124,6 @@ func RunDelete(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 	}
 
 	ignoreNotFound := cmdutil.GetFlagBool(cmd, "ignore-not-found")
-	if deleteAll {
```

Comment:
No, it should list, followed by an iterating delete, ignoring not founds on the delete by default